src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,100 @@
+package parser;
+
+import car.Car;
+import car.CarList;
+import customer.Customer;
+import customer.CustomerList;
+import transcation.Transaction;
+import transcation.TransactionList;
+import exceptions.CliRentalException;
+
+
+import java.util.Scanner;
+
+public class Parser {
+
+    public static Scanner scanner = new Scanner(System.in);
+    private static final String HELP_COMMAND = "help";
+    private static final String ADD_CUSTOMER_COMMAND = "add-user";
+    private static final String REMOVE_CUSTOMER_COMMAND = "remove-user";
+    private static final String LIST_USERS_COMMAND = "list-users";
+    private static final String ADD_CAR_COMMAND = "add-car";
+    private static final String REMOVE_CAR_COMMAND = "remove-car";
+    private static final String ADD_TRANSACTION_COMMAND = "add-tx";
+    private static final String LIST_CARS_COMMAND = "list-cars";
+    private static final String REMOVE_TRANSACTION_COMMAND = "remove-tx";
+    private static final String LIST_ALL_TRANSACTIONS = "list-tx";
+    private static final String EXIT_COMMAND = "exit";
+
+    public static String getUserInput(){
+        System.out.println("What would you like to do?");
+        System.out.println("_".repeat(60));
+
+        String userInput = scanner.nextLine().trim();
+
+        return userInput;
+    }
+
+    public static boolean parse(String userInput) throws CliRentalException {
```

Comment:
Boolean methods should start with is/has/etc.
Consider using more exceptions and asserts to assist with making sure the method runs successfully.